VazhovIA/lab1/calculator/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main()
+{
+    float left, right, result;
+    char sign;
+    cin >> left >> sign >> right;
+
+    switch (sign)
+    {
+    case '+':
+        result = left + right;
+        break;
+    case '-':
+        result = left - right;
+        break;
+    case '*':
+        result = left * right;
+        break;
+    case '/':
+        if (right == 0) {
+            cout << "Division by zero is forbidden" << endl;
+            return 8;
+        }
+        result = left / right;
+        break;
+    default:
+        cout << "wrong sign" << endl;
+        break;
+    }
+    cout << result << endl;
```

Comment:
Still wrong as the user is able to see the result output even if he entered wrong sign